Earthfile

```
@@ -427,11 +427,12 @@ iso:
     ARG ISO_NAME=${OS_ID}
     ARG IMG=docker:$IMAGE
     ARG overlay=overlay/files-iso
+    ARG TARGETARCH
     FROM $OSBUILDER_IMAGE
     WORKDIR /build
     COPY . ./
     COPY --keep-own +image-rootfs/rootfs /build/image
-    RUN /entrypoint.sh --name $ISO_NAME --debug build-iso --squash-no-compression --date=false dir:/build/image --overlay-iso /build/${overlay} --output /build/
+    RUN /entrypoint.sh --name $ISO_NAME --debug build-iso --arch $TARGETARCH --bootloader-in-rootfs --squash-no-compression --date=false dir:/build/image --overlay-iso /build/${overlay} --output /build/
```

```suggestion
    RUN /entrypoint.sh --name $ISO_NAME --debug build-iso --arch $TARGETARCH --bootloader-in-rootfs --squash-no-compression --date=false dir:/build/image --overlay-iso /build/${overlay} --output /build/
```
uh, we don't have `bootloader-in-rootfs`, not a fan of it tbh, it leaves too much room for disambiguity, better accept a folder with the bootloader - also.. osbuilder should have all the bits to create an iso and do not demand anything on the rootfs (why the rootfs would even care?) and we also cover airgap scenarios in one pass

---

ah shit, sorry this is a leftover of my local tests! Tried to generate the isos in a few different ways as the TARGETARCH wasnt working for me...dur not adding the ARG argument :D

---

ahhh this is a draft! I was wondering why I did sent this to review...I didnt :D